iothub/service/src/Messaging/MessagingClient.cs

```diff
@@ -80,7 +62,24 @@ protected MessagingClient()
         }
 
         /// <summary>
-        /// Open this instance. Must be done before any cloud-to-device messages can be sent.
+        /// The callback to be executed when the connection is lost.
```

Comment:
When would we uncomment this code? Could maybe benefit with an explanation of that scenario -- kind of subjective though.